src/pages/kb/data-sources/amazon-athena-setup.md

```diff
@@ -57,16 +71,18 @@ buckets, but please note that we have separate permissions for the bucket (
 In Redash, in the New Data Source page select "Athena" as the data source type
 and fill out the details using the information from the previous step:
 
-  * AWS Access Key and AWS Secret Key are the ones from the previous step.
-  * AWS Region is the region where you use Amazon Athena.
-  * S3 Staging Path is the bucket Amazon Athena uses for staging/query results, you might have created it already if you used Amazon Athena from AWS console - simply copy the same path.
+  * **AWS Access Key** and **AWS Secret Key** are the ones from the previous step.
+  * **AWS Region** is the region where you use Amazon Athena.
+  * **S3 Staging Path** is the bucket Amazon Athena uses for staging/query results, you might have created it already if you used Amazon Athena from AWS console - simply copy the same path.
 
 ![](/assets/images/docs/gitbook/athena_data_source.png)
 
-## Run a Query
+{% callout info %}
+
+If you have trouble refreshing the Athena data source schema, toggle **Use Glue Data Catalog** under the Additional Settings menu.
```

Comment:
Should be more like "you can try toggling" or "it might be that you're using Glue Data Catalog and need to enable it".